app/views/bikes/show.html.erb

```diff
@@ -39,29 +38,34 @@
     <div class="inner d-flex">
       <div class="details p-3 m-3">
 
-        <h6 class="avail"><% a = "Bike is available" %></h6>
-        <h6 class="unavail"><% b = "Bike is unavailable" %>
+        <h6 style="color:green"><% a = "Bike is available" %>
+        <h6 style="color:red"><% b = "Bike is unavailable" %>
 
         <%= @bike.available == true ? a: b %>
```

Comment:
Love this man, brilliant thought putting in a conditional for whether bike is available!